app/controllers/calculators_controller.rb

```diff
@@ -0,0 +1,2 @@
+class CalculatorsController < GuidesController
```

Comment:
Context of the Guide that the calculator is associated with. This allows us to display the title, breadcrumbs etc and keeps things consistent should the markdown change.